src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -569,6 +569,21 @@ object Config : Vigilant(
     )
     var changeHurtColorOnWitherKingsDragons = false
 
+    @Property(
+        type = PropertyType.SWITCH, name = "Quiver Display",
+        description = "Displays the amount of arrows in your quiver.",
+        category = "Dungeons", subcategory = "Quality of Life"
+    )
+    var showArrowsInQuiver = false
+
+    @Property(
+        type = PropertyType.NUMBER, name = "Restock Arrows Warning",
+        description = "Shows a warning when your quiver is low on arrows. Set to 0 to disable.",
+        category = "Dungeons", subcategory = "Quality of Life",
+        min = 0, max = 1500, increment = 100
```

Comment:
1400 is about half of 2880 (the max arrow count) so maybe use that

Reply:
Also, should I make a separate config for an annoying mode which will warn every warp?